docs/guides/typescript.md

```diff
@@ -71,6 +71,229 @@ Providing the context and events as generic parameters for the `createMachine()`
 - The event type (`TEvent`) ensures that only specified events (and built-in XState-specific ones) are used in transition configs. The provided event object shapes are also passed on to actions, guards, and services.
 - Events which you send to the machine will be strongly typed, offering you much more confidence in the payload shapes you'll be receiving.
 
+## The VSCode Extension <Badge text="4.26+" />
```

Comment:
The header here seems to focus on a slightly wrong thing - I would expect a typegen approach to be mentioned here and then the paragraph below could go into mentioning that VScode it's currently the only way to get this working.

Reply:
At the moment for XState itself there is no special TS version required for this feature but I wonder if it wouldn't be OK to mention that we recommend using at least TS 4.1 or something. The `@xstate/react` will require TS 4.0 for this to work.